test/new-e2e/go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	// `TEST_INFRA_DEFINITIONS_BUILDIMAGES` matches the commit sha in the module version
 	// Example: 	github.com/DataDog/test-infra-definitions v0.0.0-YYYYMMDDHHmmSS-0123456789AB
 	// => TEST_INFRA_DEFINITIONS_BUILDIMAGES: 0123456789AB
-	github.com/DataDog/test-infra-definitions v0.0.0-20231017144910-d2510f65b6fe
+	github.com/DataDog/test-infra-definitions v0.0.0-20231020114206-b6e2c8b2a497
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
It's probably in the plan, adding here as a reminder to update `TEST_INFRA_DEFINITIONS_BUILDIMAGES`

Reply:
Yes was in the plan, I updated both the module and the image with the latest commit from `test-infra-definitions` main branch